.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "boost-math"]
 	path = boost-math
 	url = https://github.com/boostorg/math.git
+[submodule "benchmarks/google-benchmark"]
+	path = benchmarks/google-benchmark
```

Comment:
We should tell `validate` to skip the submodule: https://github.com/microsoft/STL/blob/17fde2cbab6e8724d81c9555237c9a623d7fb954/tools/validate/validate.cpp#L204-L213
(This is probably passing checks because the CI doesn't check out the submodule, but `validate` should succeed if run locally on a machine with the submodule checked out.)

Note: This could be found by searching for all mentions of an existing submodule. When adding something new, looking for existing mentions is a good way to find places that should be updated.